modules/ossm-defining-namespace-receive-sidecar-injection-cluster-wide-mesh-console.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-deployment-models.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-defining-namespace-receive-sidecar-injection-cluster-wide-mesh-console_{context}"]
+= Defining which namespaces receive sidecar injection in a cluster-wide mesh using the web console
+
+Use member selectors to identify which namespaces receive sidecar injection by applying the `istio-injection: enabled` annotation.
+
+[NOTE]
+====
+Using discovery selectors to determine which namespaces the mesh can discover has no effect on sidecar injection.
+====
+
+.Prerequisites
+
+* You have installed the {SMProductName} Operator.
+* You have deployed a `ServiceMeshControlPlanae` resource and created a `ServiceMeshMemberRoll` resource.
+* You are logged in as a user with the `cluster-admin` role. If you use {product-dedicated}, you are logged in as a user with the `dedicated-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Click the {SMProductName} Operator.
+
+. Click *Istio Service Mesh Member Roll*.
+
+. Click the `ServiceMeshMemberRoll` resource.
+
+. Click *YAML*.
+
+. Modify the `spec.memberSelectors` field in the `ServiceMeshMemberRoll` by adding the `istio-injection: enabled` annotation.
```

Comment:
by adding a member selector that matches on the `istio-injection` label

Reply:
done